.github/workflows/unittests.yml

```diff
@@ -56,14 +56,15 @@ jobs:
         run: tox -e mypy
 
   utscapy:
-    name: ${{ matrix.os }} ${{ matrix.python }} ${{ matrix.mode }}
+    name: ${{ matrix.os }} ${{ matrix.installmode }} ${{ matrix.python }} ${{ matrix.mode }}
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 25
+    timeout-minutes: 20
```

Comment:
Why are you decreasing this value?

Reply:
I increased it very recently as a workaround for GLMAN tests failing, but it shouldn't be required anymore. You can have a look at the timeouts of this build: we're well under that limit.